enricher/standard/src/test/java/io/fabric8/maven/enricher/standard/MavenIssueManagementEnricherTest.java

```diff
@@ -0,0 +1,137 @@
+/*
+ *    Copyright (c) 2016 Red Hat, Inc.
+ *
+ *    Red Hat licenses this file to you under the Apache License, version
+ *    2.0 (the "License"); you may not use this file except in compliance
+ *    with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ *    implied.  See the License for the specific language governing
+ *    permissions and limitations under the License.
+ */
+
+package io.fabric8.maven.enricher.standard;
+
+import io.fabric8.maven.enricher.api.EnricherContext;
+import io.fabric8.maven.enricher.api.Kind;
+import mockit.Expectations;
+import mockit.Mocked;
+import mockit.integration.junit4.JMockit;
+import org.apache.maven.model.IssueManagement;
+import org.apache.maven.model.Scm;
+import org.apache.maven.project.MavenProject;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.Map;
+
+import static junit.framework.TestCase.*;
+
+/**
+ * @author kameshs
+ */
+@RunWith(JMockit.class)
+public class MavenIssueManagementEnricherTest {
+
+    @Mocked
+    private EnricherContext context;
+
```

Comment:
just a hint: to make you live easier you could even mock down until the issue management itself:

```java
@Mocked 
IssueManagement issueManagement
....
new Expectactions() {{
   context.getProject();
   project.getIssueManagement();
   issueManagement.getUrl(); result = "...";
   issueManagement.getSystem();  result = "..."
}}
```